hw9/hw1.py

```diff
@@ -0,0 +1,38 @@
+"""
+Write a function that merges integer from sorted files and returns an iterator
+
+file1.txt:
+1
+3
+5
+
+file2.txt:
+2
+4
+6
+
+>>> list(merge_sorted_files(["file1.txt", "file2.txt"]))
```

Comment:
does this example works?

Reply:
yes 
It runs as :
print(list(opener(["file1.txt", "file2.txt"])))